guillotina/api/registry.py

```diff
@@ -98,21 +98,13 @@ class Read(Service):
     context=IContainer, method='POST',
     permission='guillotina.RegisterConfigurations', name='@registry',
     summary='Register a new interface to for registry settings',
+    validate=True,
     parameters=[{
```

Comment:
Reminder, in open api 3, describing request bodies changed: https://swagger.io/docs/specification/describing-request-body/

Reply:
for OpenAPI change I am working on another PR(https://github.com/plone/guillotina/pull/515/files). you want me put all the change for OpenAPI in this PR only?

Reply:
Yes, let's merge put these together since they depend on one another now.